MaterialLib/SolidModels/CreateViscoElasticBurgers.h

```diff
@@ -0,0 +1,168 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PHYSICALQUANITIES_SOLIDS_CREATEVISCOELASTICBURGERS_H_
+#define PHYSICALQUANITIES_SOLIDS_CREATEVISCOELASTICBURGERS_H_
+
+#include <logog/include/logog.hpp>
+
+#include "MechanicsBase.h"
+#include "ProcessLib/Utils/ProcessUtils.h"  // required for findParameter
+#include "ViscoElasticBurgers.h"
+
+namespace Solids
+{
+template <int DisplacementDim>
+std::unique_ptr<MechanicsBase<DisplacementDim>> createViscoElasticBurgers(
+    std::vector<std::unique_ptr<ProcessLib::ParameterBase>> const& parameters,
+    BaseLib::ConfigTree const& config)
+{
+    config.checkConfigParameter("type", "ViscoElasticBurgers");
```

Comment:
Right. I'd suggest "Lubby2", as we implement the non-linear dependencies according to that model.